lib/zclient.c

```diff
@@ -1101,6 +1109,19 @@ int zapi_route_decode(struct stream *s, struct zapi_route *api)
 	return 0;
 }
 
+bool zapi_route_notify_decode(struct stream *s, struct prefix *p,
+			      enum zapi_route_notify_owner *note)
+{
+	stream_get(note, s, sizeof(*note));
+
+	p->family = stream_getc(s);
+	p->prefixlen = stream_getc(s);
+	stream_get(&p->u.prefix, s,
+		   PSIZE(p->prefixlen));
+
+	return true;
```

Comment:
Shouldn't we use `STREAM_GET()` instead of `stream_get()` and return `false` in the case of an error? We are always returning `true` here.

Reply:
Yeah so the STREAM_GET functionality was not in when I first submitted this code change.  I'll update this code before I resubmit here shortly.